### ISSUE TITLE
Futur of Uptime

### ISSUE THREAD

**Issue by gabrielstuff** [gunmetal313-nodejs/uptime2#5]
Hello there,

As I see you are currently following the steps of Francois Zaninotto to continue the project.
I've seen that @deathowl and @letrungkien0210 also forked your project. 

Original uptime has been forked and modified by more than 600 people. It is a nice number and shows some real interest.

What I would like to know, is would you be interested in opening uptime as an organization and create a clear Roadmap for uptime2.

I think that for instance security is a concern and the repo should be first updated to latest library and we get it fixed. As one of the issue mention it : https://snyk.io/test/github/fzaninotto/uptime

Also, some few improvement like user management could be a great progress

Finally the graphical part could be well improved.

Then we could also make it clear with a review of the 99 issues that the uptime users have made and see what could be included.

What are your thoughts on this ? 

Thanks !

**Comment by gunmetal313** [comment]
Hello

It good idea.Uptime is a good project for monitoring. We can discuss this and create an organization in github.

**Comment by darkweasel77** [comment]
Did the org ever get created?

**Comment by gabrielstuff** [comment]
Yes here: https://github.com/uptime2/uptime2 but for now it has not been very active.
